autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -157,6 +158,10 @@ def get_preset_models_regression(path, problem_type, objective_func, stopping_me
         if 'GBM' in custom_options:
             models += [LGBModel(path=path, name='LightGBMRegressorCustom', problem_type=problem_type, objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=get_param_baseline_custom(problem_type))]
         # SKLearnModel(path=path, name='DummyRegressor', model=DummyRegressor(), problem_type=problem_type, objective_func=objective_func),
+    if lr_options is not None:
```

Comment:
Order matters in the presets, the order it is appended is the order it is trained. I think we should have LR go directly after XT. (Both classification and regression